File: SearchAndFind-Frontend/app/src/main/java/uy/com/searchandfind/user/UserDTO.java
package uy.com.searchandfind.user;

import java.util.Collection;

import uy.com.searchandfind.Response;
import uy.com.searchandfind.saler.SalerCategoryDTO;

/**
 * Created by strat on 30/09/2017.
 */

public class UserDTO {
    private String Id ;
    private String DeviceId ;
    private String Name ;
    private String LastName ;
    private String MailAddress ;
    private String CurrentToken ;
    public UserDTO(){

    }
    public String getDeviceId() {
        return DeviceId;
    }

    public void setDeviceId(String aDeviceid) {
        DeviceId = aDeviceid;
    }


    public String getId() {
        return Id;
    }

    public void setId(String id) {
        Id = id;
    }
    public String getName() {
        return Name;
    }

    public void setName(String name) {
        Name = name;
    }

    public String getLastName() {
        return LastName;
    }

    public void setLastName(String lastName) {
        LastName = lastName;
    }

    public String getMailAddress() {
        return MailAddress;
    }

    public void setMailAddress(String mailAddress) {
        MailAddress = mailAddress;
    }

    public String getCurrentToken() {
        return CurrentToken;
    }

    public void setCurrentToken(String currentToken) {
        CurrentToken = currentToken;
    }
}

File: SearchAndFindBackend/SearchAndFind.DTO/UserDTO.cs
using System;

namespace SearchAndFind.DTO
{
    public class UserDTO
    {
        public Guid Id { get; set; }
        public string DeviceId { get; set; }
        public string Name { get; set; }
        public string LastName { get; set; }
        public string MailAddress { get; set; }
        public string CurrentToken { get; set; }
        public int NumberOfReview { get; set; }
        public double AverageReview { get; set; }
        public UserDTO() { }
        public UserDTO(Guid id, string name, string lastName, string mailAddress, string deviceId)
        {
            Id = id;
            Name = name;
            LastName = lastName;
            MailAddress = mailAddress;
            DeviceId = deviceId;
        }
    }
}
